nginx.conf

```diff
@@ -6,6 +6,7 @@ events {
 }
 
 http {
+    access_log    /dev/stdout;
```

Comment:
Does this need separate logs for error/access? (e.g. the official nginx image uses a symlink for this https://github.com/nginxinc/docker-nginx/blob/e950fa7dfcee74933b1248a7fe345bdbc176fffb/mainline/jessie/Dockerfile#L21-L23)